### DISCUSSION TITLE
Area discard pixel - box

### DISCUSSION THREAD

**Discussion by marioepsley** [cables-gl/cables_docs#418] [! Welcome - readme first!]
AreaDiscardPixel
Such a useful op.
 
the Box setting doesn't work correctly.

**Comment by steam0r** [comment]
can you provide an example or describe what is wrong/expected? thanks!

**Reply by marioepsley** [reply]
I was imagining it working just the same as the sphere
https://cables.gl/p/kOBf3j

if you change the area to box in this example, then adjust the area discards size, you'll notice it doesn't quite work in the same way. If you start at 0 and slowly slide it to 2, you'll see it kind of works, any values after 2 it just snaps out.
